hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -810,6 +810,41 @@ message KeyLocationList {
     optional bool isMultipartKey = 4 [default = false];
 }
 
+/**
+ * Checksum algorithms/types used in Ozone
+ * Make sure this enum's integer values match enum values' id properties defined
+ * in org.apache.hadoop.util.DataChecksum.Type
+ */
+enum ChecksumTypeProto {
+  CHECKSUM_NULL = 0;
+  CHECKSUM_CRC32 = 1;
+  CHECKSUM_CRC32C = 2;
+}
+
+enum FileChecksumTypeProto {
+  MD5CRC = 1;  // BlockChecksum obtained by taking the MD5 digest of chunk CRCs
+  COMPOSITE_CRC = 2;  // Chunk-independent CRC, optionally striped
+}
+
+message CompositeCrcFileChecksumProto {
+  required ChecksumTypeProto checksumType = 1;
+  required uint32 bytesPerCrc = 2;
+  required uint32 crc = 3;
+}
+
+message MD5MD5Crc32FileChecksumProto {
+  required ChecksumTypeProto checksumType = 1;
+  required uint32 bytesPerCRC = 2;
+  required uint64 crcPerBlock = 3;
+  required bytes md5 = 4;
+}
+
+message FileChecksumProto {
+  required FileChecksumTypeProto checksumType = 1;
```

Comment:
Can you add test cases to cover `FileChecksumTypeProto + ChecksumTypeProto` combinations.

Reply:
could you elaborate? Didn't quite understand. Do you mean add unit test to cover OMPBHelper.convert() APIs?

Reply:
Yes, exactly. 

```
MD5CRC + CHECKSUM_CRC32
MD5CRC + CHECKSUM_CRC32C
COMPOSITE_CRC + CHECKSUM_CRC32
COMPOSITE_CRC + CHECKSUM_CRC32C 

+Probably all the null cases...
```

Reply:
@jojochuang  I'm OK to raise a separate unit test task and cover all these tests in that. Please create a sub-task, then we can merge this PR and move ahead to the next task.

Reply:
Filed https://issues.apache.org/jira/browse/HDDS-5996 as the placeholder. Thx